FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -211,15 +228,15 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 <locus from="127r" to="127v">No. 15</locus> being a miracle of ʾEwosṭātewos and <persName ref="PRS10048Warasin"></persName>
                                 similar to <bibl><ptr target="bm:ContiRossini1912Studi"/><citedRange unit="page">633-634</citedRange></bibl>
                                 but longer than that with an additional note on an encounter with a hermit.</note>
-                            <colophon xml:id="coloph2"><locus from="122ra" to="122rb" xml:lang="gez">ተጽሕፈት፡ ዛቲ፡ መጽሐፍ፡ በዘመነ፡ ዮሐንስ፡ 
+                            <colophon xml:id="coloph2"><locus from="122ra" to="122rb" xml:lang="gez"/>ተጽሕፈት፡ ዛቲ፡ መጽሐፍ፡ በዘመነ፡ ዮሐንስ፡ 
                                 ወንጌላዊ፡ በ፸፻ወ፪፻፡ ፳ወ፰፡ ዓመተ፡ ዓለም፡ ወሠርቀ፡ ሌሊት፡ ፬፡ ወሠርቀ፡ መዓልት፡ ፳ወ፮፡ በዘመነ፡ ንጉሥነ፡ <persName ref="PRS2377Bakaffa">በካፋ፡</persName> 
-                                ወጳጳስነ፡ <persName ref="PRS6072Krestodo">ክርስቶዶሉ፡</persName> ወመምሕርነሂ፡ <persName>መቃርዮስ፡</persName> 
-                                ወቂሰ፡ ገበዝሂ፡ <persName>በዕዳ፡ ማርያም፡</persName> ወጸሐፊሃኒ፡ <persName>አርሳንዮስ፡</persName> ዘተድባበ፡ በመንግሥተ፡ ሰማያት፡ አሜን። 
-                                ዛቲ፡ መጽሐፍ፡ ተወጽነት፡ አመ፡ ረቡዑ፡ ለሐምሌ፡ ወተፈጸመት፡ በካልአይ፡ አመት፡ አመ፡ ፳ሀ፡ ለሕዳር፡ በዘመነ፡ <persName>ማቴዎስ</persName>።</locus></colophon>
-                            <msItem/>
+                                ወጳጳስነ፡ <persName ref="PRS6072Krestodo">ክርስቶዶሉ፡</persName> ወመምሕርነሂ፡ <persName ref="PRS00000">መቃርዮስ፡</persName> 
```

Comment:
`persName `MUST have a `ref ` which must have a link. or do not add the `persName `tag at all